xio-core/src/main/java/com/xjeffrose/xio/config/RouteReloader.java

```diff
@@ -0,0 +1,169 @@
+package com.xjeffrose.xio.config;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static com.google.common.base.Preconditions.checkState;
+
+import com.google.common.annotations.VisibleForTesting;
+import java.io.*;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.security.MessageDigest;
+import java.util.*;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+import java.util.function.BiConsumer;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class RouteReloader<T> {
+
+  private static class RouteMeta<T> {
```

Comment:
static inner and inner classes usually are ordered to the bottom of the parent class

Reply:
ok will move